File: examples/sunnah/chapters/node/listTheChaptersInABook.js
const { SunnahService } = require("m3o/sunnah");

// Get all the chapters of a given book within a collection.
async function listTheChaptersInAbook() {
  let sunnahService = new SunnahService(process.env.MICRO_API_TOKEN);
  let rsp = await sunnahService.chapters({
    book: 1,
    collection: "bukhari",
  });
  console.log(rsp);
}

listTheChaptersInAbook();

File: examples/go/sunnah/chapters/listTheChaptersInABook.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/sunnah"
)

// Get all the chapters of a given book within a collection.
func ListTheChaptersInAbook() {
	sunnahService := sunnah.NewSunnahService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := sunnahService.Chapters(&sunnah.ChaptersRequest{
		Book:       1,
		Collection: "bukhari",
	})
	fmt.Println(rsp, err)
}
